### DISCUSSION TITLE
pfe-tabs with image

### DISCUSSION THREAD

**Discussion by phillip-kruger** [patternfly/patternfly-elements#2127] [Q&A] [answered]
I want to add an image to my pfe-tab (inside the pfe-tabs). Is this something you will consider adding ?

**Comment by bennypowers** [comment, marked as answer]
do you mean in the tab header or do you mean in the content?

PatternFly tabs can have icons, so we could should support that too

https://www.patternfly.org/v4/components/tabs/#icons-and-text

**Reply by phillip-kruger** [reply]
Yea the header, exactly like the v4 example

**Reply by bennypowers** [reply]
In our [draft PR for pfe-tabs v2](https://github.com/patternfly/patternfly-elements/pull/2099#issuecomment-1228333696), I've liked this discussion, to make sure we implement icon support.

pfe-ifon itself will also be getting a rewrite, mind.

**Reply by zeroedin** [reply]
The current implementation is subject to change before release but here is the general idea.  When rewritten, `pfe-icon` and SVG's will be supported in an `icon` slot.

```html
<pfe-tabs>
  <pfe-tab slot="tab"> 
    <svg slot="icon">...</svg> 
    <!-- or -->
    <pfe-icon slot="icon"></pfe-icon>
    Tab text
  </pfe-tab>
  <pfe-tab-panel>Panel content</pfe-tab-panel>
  <pfe-tab slot="tab"> ... </pfe-tab>
  <pfe-tab-panel> ... </pfe-panel>
 </pfe-tabs>
 ```
